python/lib/monitor/dmod/monitor/que_monitor.py

```diff
@@ -1,32 +1,14 @@
 #!/usr/bin/env python3
```

Comment:
should probably version bump the monitor package with this

Reply:
Yeah, I'm adjusting the version of this and several of the other packages.  I think I did that at the beginning (so I wouldn't forget) not quite anticipating the significance of the changes to come.  Will commit that shortly.